src/lib/coda_version/coda_version_defaults.ml

```diff
@@ -0,0 +1,12 @@
+let commit_id = "0000000000000000000000000000000000000000"
```

Comment:
This is failing the formatting check on CI; can you run `make reformat` and check in the generated formatting changes?

Reply:
That did not seem to change anything.  But I just realized I think I need to add `(modules coda_version)` to the dune file, don't I?  `coda_version_default.ml` is just for Bazel.

Reply:
It should be fine as-is, dune will compile it but it won't get linked because it's never referenced, I believe.

Reply:
reformat is done and pushed. should I revert the change to the dune file?

Reply:
It's fine, lets leave it